test/e2e/specs/demo-content-with-keyboard.test.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * Node dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import uuid from 'uuid/v4';
+
+/**
+ * Internal dependencies
+ */
+import {
+	newPost,
+	pressWithModifier,
+	META_KEY,
+	pressTimes,
+	//getEditedPostContent,
+} from '../support/utils';
+
+async function uploadImageInTheMediaLibrary( assetFileName ) {
+	await page.waitForSelector( '.media-modal input[type=file]' );
+	const inputElement = await page.$( '.media-modal input[type=file]' );
+	const testImagePath = path.join( __dirname, '..', 'assets', assetFileName );
+	const filename = uuid();
+	const tmpFileName = path.join( os.tmpdir(), filename + '.png' );
+	fs.copyFileSync( testImagePath, tmpFileName );
+	await inputElement.uploadFile( tmpFileName );
+	await page.waitForSelector( `.media-modal li[aria-label="${ filename }"]` );
+}
+
+async function navigateToNextRegion() {
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.down( 'Shift' );
+	await page.keyboard.type( 'n' );
+	await page.keyboard.up( 'Shift' );
+	await page.keyboard.up( 'Alt' );
+}
+
+async function createBlockAfter() {
+	await page.keyboard.down( META_KEY );
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.type( 'y' );
+	await page.keyboard.up( 'Alt' );
+	await page.keyboard.up( META_KEY );
+}
+
+jest.setTimeout( 1000000 );
```

Comment:
I assume this is for the upload, but worth adding a comment to explain 😄

Reply:
Actually, it's because this test take too long since it's a very long post :P (I think our current limit is 10 seconds)

Reply:
Hah! Right, that makes sense.

Reply:
Travis doesn't like it:

> waiting for selector ".media-modal input[type=file]" failed: timeout 30000ms exceeded